source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.h

```diff
@@ -63,7 +63,7 @@ class SwiftASTManipulatorBase {
 
     VariableInfo(CompilerType &type, swift::Identifier name,
                  VariableMetadataSP metadata, 
-                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Let,
+                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Var,
```

Comment:
Why are you defining a variable in order to pass a constant to this constructor?